app/src/utils.js

```diff
@@ -1,7 +1,39 @@
+/* eslint-disable max-statements */
+import { validator } from '@liskhq/lisk-client';
+import { requestTokenSchema } from './validationSchema';
+
 const PERMISSION_WHITE_LIST = ['clipboard-read', 'notifications', 'openExternal'];
+const WHITE_LISTED_DEEP_LINKS = [
+  {
+    pathRegex: /^\/\/wallet\/?$/,
+    validationSchema: requestTokenSchema,
+  },
+];
 
 export const setRendererPermissions = (win) => {
   win.browser.webContents.session.setPermissionRequestHandler((_, permission, callback) => {
     callback(PERMISSION_WHITE_LIST.includes(permission));
   });
 };
+
+export const canExecuteDeepLinking = (url) => {
```

Comment:
Should it also clean invalid url params? Or completely reject those url? For example
`open "lisk://wallet?recipient=lskdb3ja5dhr25jhwm6onstawj33yk74g8q2549oc&amount=2300&token=0200000000000000&recipientChain=02000000&modal=send&name=oskar"`

Should that:
1. Dont open the send modal bc of url param `name=oskar`
2. Open send modal but clean the url from `name=oskar`
3. Behave like this PR does now, opens the send modal and keeps the `name=oskar` param

Reply:
done

Reply:
Nice